deploy.bash

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# No failures allowed
```

Comment:
![failure is always an option](https://miro.medium.com/max/607/1*aCfvyxPjVu1RDkewpy52wA.jpeg)

Reply:
:joy: